rotors_gazebo_plugins/include/rotors_gazebo_plugins/gazebo_pressure_plugin.h

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2016 Pavel Vechersky, ASL, ETH Zurich, Switzerland
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef ROTORS_GAZEBO_PLUGINS_PRESSURE_PLUGIN_H
+#define ROTORS_GAZEBO_PLUGINS_PRESSURE_PLUGIN_H
+
+#include <gazebo/common/Plugin.hh>
+#include <gazebo/common/common.hh>
+#include <gazebo/gazebo.hh>
+#include <gazebo/physics/physics.hh>
+
+#include "FluidPressure.pb.h"
+
+#include "rotors_gazebo_plugins/common.h"
+
+namespace gazebo {
+// Constants
+static constexpr double kRs = 8314.32; /* Nm/ (kmol K), gas constant */
+static constexpr double kM0 = 28.9644; /* kg/kmol, mean molecular weight of air */
+static constexpr double kG0 = 9.80665; /* m/s^2, acceleration due to gravity at 45.5425 deg lat */
+static constexpr double kR0 = 6356766.0; /* m, Earth radius at g0 */
+static constexpr double kP0 = 101325.0; /* Pa, air pressure at g0 */
+static constexpr double kT0 = 288.15; /* K, standard sea-level temperature */
+static constexpr double kTl = 0.0065; /* K/m, temperature lapse */
+static constexpr double kAs = kG0 * kM0 / (kRs * -kTl);
```

Comment:
Agree. @pvechersky Maybe just use the comments as variable names (incl. units) and remove comments.